<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"><head><title>TestNG Report</title><style type="text/css">table {margin-bottom:10px;border-collapse:collapse;empty-cells:show}th,td {border:1px solid #009;padding:.25em .5em}th {vertical-align:bottom}td {vertical-align:top}table a {font-weight:bold}.stripe td {background-color: #E6EBF9}.num {text-align:right}.passedodd td {background-color: #3F3}.passedeven td {background-color: #0A0}.skippedodd td {background-color: #DDD}.skippedeven td {background-color: #CCC}.failedodd td,.attn {background-color: #F33}.failedeven td,.stripe .attn {background-color: #D00}.stacktrace {white-space:pre;font-family:monospace}.totop {font-size:85%;text-align:center;border-bottom:2px solid #000}</style></head><body><table><tr><th>Test</th><th># Passed</th><th># Skipped</th><th># Failed</th><th>Time (ms)</th><th>Included Groups</th><th>Excluded Groups</th></tr><tr><th colspan="7">RetryFailedTestSuite</th></tr><tr><td><a href="#t0">RetryMulitpleFailedTestCases</a></td><td class="num">0</td><td class="num attn">9</td><td class="num attn">3</td><td class="num">212</td><td></td><td></td></tr></table><table><thead><tr><th>Class</th><th>Method</th><th>Start</th><th>Time (ms)</th></tr></thead><tbody><tr><th colspan="4">RetryFailedTestSuite</th></tr></tbody><tbody id="t0"><tr><th colspan="4">RetryMulitpleFailedTestCases &#8212; failed</th></tr><tr class="failedeven"><td rowspan="3">com.testcases.RetryLogicTest</td><td><a href="#m0">test1</a></td><td rowspan="1">1565433513382</td><td rowspan="1">4</td></tr><tr class="failedeven"><td><a href="#m1">test2</a></td><td rowspan="1">1565433513422</td><td rowspan="1">4</td></tr><tr class="failedeven"><td><a href="#m2">test3</a></td><td rowspan="1">1565433513454</td><td rowspan="1">0</td></tr><tr><th colspan="4">RetryMulitpleFailedTestCases &#8212; skipped</th></tr><tr class="skippedeven"><td rowspan="9">com.testcases.RetryLogicTest</td><td><a href="#m3">test1</a></td><td rowspan="3">1565433513274</td><td rowspan="3">52</td></tr><tr class="skippedeven"><td><a href="#m4">test1</a></td></tr><tr class="skippedeven"><td><a href="#m5">test1</a></td></tr><tr class="skippedeven"><td><a href="#m6">test2</a></td><td rowspan="3">1565433513398</td><td rowspan="3">4</td></tr><tr class="skippedeven"><td><a href="#m7">test2</a></td></tr><tr class="skippedeven"><td><a href="#m8">test2</a></td></tr><tr class="skippedeven"><td><a href="#m9">test3</a></td><td rowspan="3">1565433513446</td><td rowspan="3">0</td></tr><tr class="skippedeven"><td><a href="#m10">test3</a></td></tr><tr class="skippedeven"><td><a href="#m11">test3</a></td></tr></tbody></table><h2>RetryMulitpleFailedTestCases</h2><h3 id="m0">com.testcases.RetryLogicTest#test1</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [false] but found [true]
	at com.testcases.RetryLogicTest.test1(RetryLogicTest.java:10)
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m1">com.testcases.RetryLogicTest#test2</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [false] but found [true]
	at com.testcases.RetryLogicTest.test2(RetryLogicTest.java:15)
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m2">com.testcases.RetryLogicTest#test3</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [false] but found [true]
	at com.testcases.RetryLogicTest.test3(RetryLogicTest.java:20)
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m3">com.testcases.RetryLogicTest#test1</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [false] but found [true]
	at com.testcases.RetryLogicTest.test1(RetryLogicTest.java:10)
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m4">com.testcases.RetryLogicTest#test1</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [false] but found [true]
	at com.testcases.RetryLogicTest.test1(RetryLogicTest.java:10)
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m5">com.testcases.RetryLogicTest#test1</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [false] but found [true]
	at com.testcases.RetryLogicTest.test1(RetryLogicTest.java:10)
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m6">com.testcases.RetryLogicTest#test2</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [false] but found [true]
	at com.testcases.RetryLogicTest.test2(RetryLogicTest.java:15)
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m7">com.testcases.RetryLogicTest#test2</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [false] but found [true]
	at com.testcases.RetryLogicTest.test2(RetryLogicTest.java:15)
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m8">com.testcases.RetryLogicTest#test2</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [false] but found [true]
	at com.testcases.RetryLogicTest.test2(RetryLogicTest.java:15)
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m9">com.testcases.RetryLogicTest#test3</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [false] but found [true]
	at com.testcases.RetryLogicTest.test3(RetryLogicTest.java:20)
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m10">com.testcases.RetryLogicTest#test3</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [false] but found [true]
	at com.testcases.RetryLogicTest.test3(RetryLogicTest.java:20)
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p><h3 id="m11">com.testcases.RetryLogicTest#test3</h3><table class="result"><tr><th>Exception</th></tr><tr><td><div class="stacktrace">java.lang.AssertionError: expected [false] but found [true]
	at com.testcases.RetryLogicTest.test3(RetryLogicTest.java:20)
... Removed 30 stack frames</div></td></tr></table><p class="totop"><a href="#summary">back to summary</a></p></body></html>